shell/common/shell.cc

```diff
@@ -146,6 +146,8 @@ std::unique_ptr<Shell> Shell::Create(
 #if FML_OS_ANDROID || FML_OS_IOS
   if (!vm) {
     shorebird_report_launch_failure();
+  } else {
+    shorebird_report_launch_success();
```

Comment:
Is this correct? I vaguely remember us talking about wanted to wait 5s to make sure the boot was successful before reporting the launch as a success. Also isn't the event a patch install success? We may want to have a separate event for patch_launch? wdyt?

Reply:
> Is this correct? I vaguely remember us talking about wanted to wait 5s to make sure the boot was successful before reporting the launch as a success

Yeah, I mentioned in the description that we may eventually want to be more sophisticated about this, but this at least seems like a step in the right direction. 

> Also isn't the event a patch install success? 

That is the event we're reporting to the server. The updater knows whether there was a patch or not and has separate logic to mark a patch as good.

Reply:
The more I look at this, the less I like it. I think this would lead to us calling `report_launch_success` before `report_launch_start`, which isn't right. Going to move this to a draft for now.

Reply:
Dropped in some log statements, and I think this is more correct than I thought – this happens after `ConfigureShorebird`, which calls `shorebird_init` and kicks off the updater thread. Moving this to a non-draft state.